src/panels/ConfigPanel/ConfigField.tsx

```diff
@@ -0,0 +1,141 @@
+import { ReactNode, useEffect, useRef, useState } from "react";
```

Comment:
This component contains everything that was duplicated between `TextAreaConfig` and `BooleanConfig`.